prow/cluster/statusreconciler_deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
       terminationGracePeriodSeconds: 180
       containers:
       - name: statusreconciler
-        image: gcr.io/k8s-prow/statusreconciler:v20181129-31c544b
+        image: gcr.io/k8s-prow/status-reconciler:v20190109-6897efb
```

Comment:
The mixing of `statusreconciler` and `status-reconciler` throughout filenames, config files, and BUILD files is :-1: 
Lets not hold up this fix though.
/lgtm